wandb/sdk/internal/sender.py

```diff
@@ -890,6 +892,32 @@ def _setup_fork(self, server_run: dict):
         self._run.forked = True
         self._run.starting_step = first_step
 
+    def _load_rewind_state(self, run: "RunRecord"):
+        assert self._settings.resume_from
+        self._rewind_response = self._api.rewind_run(
```

Comment:
why do you need to keep this one as part of the sender class?

Reply:
because it needs to be accessed by `_init_run()` and I didn't want to change the function signature of that method.